staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -152,6 +153,15 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				options,
 			)
 		}
+		if accessor, accessorErr := meta.Accessor(obj); accessorErr == nil {
```

Comment:
Please add comment indicating that this must come after admission so that we properly de-dupe ownerreferences added by mutating admission.

Reply:
This line happens before mutating admission. I added another dedup call after mutating admission

The orders are:
- CREATE: dedup -> field manager -> admission -> dedup
- UPDATE: dedup -> (field manager -> admission -> dedup) * N
- PATCH: (apply patch -> field manager -> admission -> dedup) * N

For patch requests a bad request can still wipes out managed fields. And it's not easy to pass the request context down to each patcher.